test/e2e/helpers.js

```diff
@@ -117,6 +121,16 @@ async function withFixtures(options, testSuite) {
       contractRegistry = ganacheSeeder.getContractRegistry();
     }
 
+    if (enableWebSocketServer) {
+      webSocketServer = new WebSocket.Server({ port: 8081, host: '0.0.0.0' });
```

Comment:
did enabling websocket mocking not work (`mockServer.forAnyWebSocket()...`)?

Reply:
This is now solved with Mocking websocket requests using `await server.forAnyWebSocket().thenEcho();
`